### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by aish0508**
Done with competition task changes and share pull request again to review.

### CODE REVIEW

File: MarsQA-CompetitionTask2/Test/Certificate_Test.cs [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        {
+            loginobj = new Login();
+            profileobj = new Profile();
+            editObj = new Edit();
+            deleteObj = new Delete();
+
+
+        }
+        [OneTimeSetUp]
+        public static void ExtentStart()
```
Comment (amanbansalnz) [inline MarsQA-CompetitionTask2/Test/Certificate_Test.cs]: so something minor but it is best practice to put your oneTimeSetup in a hooks class as it executes once which is used to setup things one time for example your extent report should be declared once not in every class and then you should use the instance in classes https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html

for your advance task please make sure you fix this

File: MarsQA-CompetitionTask2/Test/Certificate_Test.cs [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            extent.AttachReporter(sparkReporter);
+        }
+
+        [SetUp]
+        public void Login()
+        {
+
+
+            //Open Chrome Browser
+            Initialize();
```
Comment (amanbansalnz) [inline MarsQA-CompetitionTask2/Test/Certificate_Test.cs]: this is not correct you should do this in your onetimesetup and not on setup can you please fix the above and this as I think you still have not understood how hooks work! Can you also fix the other test class! Ideally you should have a Hooks calls which will create the Common Driver instance which will call the Initialize method!

**Review by amanbansalnz — CHANGES_REQUESTED** [review]
So the rest looks good just need to fix your hooks so that you get an understanding of how this is working!

File: MarsQA-CompetitionTask2/Test/Certificate_Test.cs (hunk 1 above)
Comment (aish0508) [reply MarsQA-CompetitionTask2/Test/Certificate_Test.cs]: Done Changes .Please review my code now.

File: MarsQA-CompetitionTask2/Test/Certificate_Test.cs (hunk 2 above)
Comment (aish0508) [reply MarsQA-CompetitionTask2/Test/Certificate_Test.cs]: Done changes .Please review my code now.

**Pull request closed without merging**